src/witan/gwyn/utils.clj

```diff
@@ -11,3 +11,7 @@
   (if (zero? dd)
     0.0
     (/ ^double d dd)))
+
+(defn average
```

Comment:
I use it in at least two calculations. But you're right there's something wrong. I'll fix it!